versions.json

```diff
@@ -633,5 +633,11 @@
             "javaVersion": "8.0.0.beta3",
             "pro": true
         }
+    },
+    "bundles": {
+        "vaadin": {
+            "npmName": "@vaadin/bundles",
+            "jsVersion": "23.0.0-beta2"
```

Comment:
Considering about the shrinkwrap, with platform snapshot, we dont touch the shrinkwrap. shrinkwrap java file will only be updated during the platform release (while, we skip tests for releases to save time)

Reply:
Yes, snapshots builds should skip generating/testing the bundle like with shrinkwrap because we cannot publish snapshots in npm.
Let's do in the same way, otherwise we need a lot of logic keeping this version updated only for running the test here

Reply:
While we skip the shrinkwrap and npm package generation are skipped for platform snapshots, still `vaadin-core` platform snapshot has `vaadin_versions.json` with latest component npm package versions. I think we need something like that for the `VaadinBundle.java`, so that those who depend on snapshot platform `vaadin` artifact get the latest `@vaadin/bundles` npm package installed for them, not only for running the test.

Reply:
Also, we want `package.json` generation in Flow to keep version of `@vaadin/bundles` consistent with other WC. So for that matter I added the `versions.json` entry also to propagate it to `vaadin_version.json` resource.

Reply:
by the way, the current failure is because the @vaaidn/bundles 23.0.0.beta2 has not been released. https://www.npmjs.com/package/@vaadin/bundles

Reply:
Right now what I see is missing is somehow keeping this exact version up-to-date. How about (roughly) this process:
- during platform release, we generate and release `@vaadin/bundles` npm package
- then a platform release step commits the latest version to `version.json` here. (Much like it [commits](https://github.com/vaadin/vaadin/commit/797a094de8037349b5764a877c1cf070095a443c) version changes in `vaadin/vaadin-core`/`vaadin/vaadin` repos, but this time the package is hosted in the platform repo).
- next platform `vaadin` artifact snapshot has `VaadinBundles.java` with latest version updated by the previous step

Would that be possible?

Reply:
it is possible, but platform repo is different than the vaadin/vaaidn-core repos, as you can push directly to those repos directly. 
if we want to do it in that way, we probably should make the bundle release separate from platform releases. so it can create a PR to platform to update the version. than after merging the PR, we do platform release. (similar like a WC release.)

Reply:
> by the way, the current failure is because the @vaaidn/bundles 23.0.0.beta2 has not been released. > https://www.npmjs.com/package/@vaadin/bundles

thanks @ZheSun88 , I released it